reference/conanfile/tools/microsoft.rst

```diff
@@ -577,3 +577,68 @@ Example:
 
     def validate(self):
         check_min_vs(self, "192")
+
+
+
+NMakeDeps
+---------
+
+Available since: `1.55.0 <https://github.com/conan-io/conan/releases/tag/1.55.0>`_
+
+This generator can be used as:
+
+.. code-block:: python
+
+    from conan import ConanFile
+
+    class Pkg(ConanFile):
+        settings = "os", "compiler", "build_type", "arch"
+
+        requires = "mydep/1.0"
+        # attribute declaration
+        generators = "NMakeDeps"
+
+        # OR explicit usage in the generate() method
+        def generate(self):
+            deps = NMakeDeps(self)
+            deps.generate()
+
+        def build(self):
+            self.run(f"nmake /f makefile")
+
+The generator will create a ``conannmakedeps.bat`` environment script that defines
+``CL``, ``LIB`` and ``_LINK_`` environment variables, injecting necessary flags 
+to locate and link the dependencies declared in ``requires``.
```

Comment:
Is it worth saying that it should be combined with `NMakeToolchain`?

Reply:
Sure, done.